File: mtgSearch/src/main/java/com/dbottillo/mtgsearchfree/model/storage/SetsStorage.java
package com.dbottillo.mtgsearchfree.model.storage;

import com.dbottillo.mtgsearchfree.model.MTGSet;
import com.dbottillo.mtgsearchfree.model.database.MTGDatabaseHelper;
import com.dbottillo.mtgsearchfree.util.LOG;

import java.util.List;

public class SetsStorage {

    private MTGDatabaseHelper databaseHelper;

    public SetsStorage(MTGDatabaseHelper databaseHelper) {
        LOG.d("created");
        this.databaseHelper = databaseHelper;
    }


    public List<MTGSet> load() {
        LOG.d();
        return databaseHelper.getSets();
    }

}

File: mtgSearch/src/main/kotlin/com/dbottillo/mtgsearchfree/model/storage/SetsStorage.kt
package com.dbottillo.mtgsearchfree.model.storage

import android.content.Context
import com.dbottillo.mtgsearchfree.database.MTGDatabaseHelper
import com.dbottillo.mtgsearchfree.resources.MTGSet
import java.util.*

class SetsStorage(var context: Context) {

    fun load(): ArrayList<MTGSet> {
        var helper = MTGDatabaseHelper(context)
        return helper.sets
    }

}

